obspy/io/seiscomp/inventory.py

```diff
@@ -457,20 +457,18 @@ def _read_channel(instrumentation_register, cha_element, _ns):
         if len(response_elements) == 0:
             msg = ("Could not find response tag with public ID "
                    "'{}'.".format(response_id))
-            #raise obspy.ObsPyException(msg)
-            print ('{} - Response ERROR:'.format(seed_id))
-            print (msg)
-            print (' -> Omitting response information from Inventory')
-            print ('')
+            warnings.warn(msg)
+            msg = ("Omitting response information from Inventory for "
+                    "'{}'.".format(seed_id))
+            warnings.warn(msg)
             response_element = None
         elif len(response_elements) > 1:
             msg = ("Found multiple matching response tags with the same "
                    "public ID '{}'.".format(response_id))
-            #raise obspy.ObsPyException(msg)
-            print ('{} - Response ERROR:'.format(seed_id))
-            print (msg)
-            print (' -> Omitting response information from Inventory')
-            print ('')
+            warnings.warn(msg)
+            msg = ("Omitting response information from Inventory for "
+                    "'{}'.".format(seed_id))
+            warnings.warn(msg)
```

Comment:
We want to have those two messages combined into a single warning with two sentences/lines, as it is also logically only one event to message about.

Reply:
Fixed it.